client/src/main/resources/lang/messages_en_US.properties

```diff
@@ -34,4 +34,25 @@ title.main=Main Menu
 title.register=Register
 title.settings=Settings
 sceneChange.user.deleted.successfully=User deleted successfully
-sceneChange.connection.lost=Connection to server lost
\ No newline at end of file
+sceneChange.connection.lost=Connection to server lost
+title.server=Server
+connection.status.not_connected=Not Connected
+connection.status.connected=Connected
+connection.status.connecting=Connecting...
+connection.status.failed=Connection Failed
```

Comment:
I made the corrections in the German language file but now I see the capitalized letter in the English file as well. Did you do this to emphasize the action or why are they capitalized?

Reply:
I looked up the GNOME guidelines: https://developer.gnome.org/hig/guidelines/writing-style.html
They are also capitalizing more words than you would in a normal English sentence. We could adopt those rules in German as well.